lib/elixir/lib/keyword.ex

```diff
@@ -832,7 +832,7 @@ defmodule Keyword do
   """
   @spec drop(t, [key]) :: t
   def drop(keywords, keys) when is_list(keywords) do
-    :lists.filter(fn {k, _} -> not k in keys end, keywords)
+    :lists.filter(fn {k, _} -> k not in keys end, keywords)
```

Comment:
Since we're touching this line, it could be great opportunity to get rid of single-letter variable.